backend/fms_core/serializers.py

```diff
@@ -590,32 +589,48 @@ class Meta:
         fields = "__all__"
 
 class DatasetSerializer(serializers.ModelSerializer):
-    files = serializers.SerializerMethodField()
+    # files = serializers.SerializerMethodField()
     released_status_count = serializers.SerializerMethodField()
     blocked_status_count = serializers.SerializerMethodField()
-    latest_release_update = serializers.SerializerMethodField()
     readset_count = serializers.SerializerMethodField()
     archived_comments = ArchivedCommentSerializer("archived_comments", many=True)
+    latest_release_update = serializers.SerializerMethodField()
+    released_by = serializers.SerializerMethodField()
+    validation_status = serializers.SerializerMethodField()
+    validation_status_update_timestamp = serializers.SerializerMethodField()
+    validated_by = serializers.SerializerMethodField()
 
     class Meta:
         model = Dataset
-        fields = ("id", "external_project_id", "run_name", "lane", "files", "released_status_count", "blocked_status_count", "latest_release_update", "validation_status", "project_name", "metric_report_url", "readset_count", "archived_comments")
+        fields = "__all__"
```

Comment:
If you serialize more fields make sure they are added to the model in the frontend. Here released_by, validated_by, to_be_replaced(validation_status_update_timestamp) and experiment_run are not in fms_api_models.ts